go.mod

```
@@ -6,28 +6,31 @@ require (
 	github.com/Jeffail/gabs v1.4.0
 	github.com/Jeffail/gabs/v2 v2.6.1
 	github.com/ghodss/yaml v1.0.0
-	github.com/imdario/mergo v0.3.11
-	github.com/openshift/api v0.0.0-20210706092853-b63d499a70ce
-	github.com/openshift/client-go v0.0.0-20210521082421-73d9475a9142
-	github.com/openshift/library-go v0.0.0-20210720093535-f8ed43828870
+	github.com/imdario/mergo v0.3.12
+	github.com/openshift/api v0.0.0-20211217221424-8779abfbd571
+	github.com/openshift/client-go v0.0.0-20211209144617-7385dd6338e3
+	github.com/openshift/library-go v0.0.0-20211220195323-eca2c467c492
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus-operator/prometheus-operator v0.52.1
-	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.52.1
-	github.com/prometheus-operator/prometheus-operator/pkg/client v0.52.1
+	github.com/prometheus-operator/prometheus-operator v0.53.1
+	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.53.1
+	github.com/prometheus-operator/prometheus-operator/pkg/client v0.53.1
 	github.com/prometheus/client_golang v1.11.0
-	github.com/prometheus/prometheus v1.8.2-0.20211005150130-f29caccc4255 // v1.8.2 is misleading as Prometheus does not have v2 module. This is pointing to v2.26.1, the same as in prometheus-operator v0.52.1
-	golang.org/x/crypto v0.0.0-20210616213533-5ff15b29337e
+	github.com/prometheus/prometheus v2.5.0+incompatible
```

why not use the pseudo-version directly here? In general, I prefer to use `replace` only for dependencies that can't be resolved properly by `go mod` (like k8s.io/client-go in this file).

```suggestion
	github.com/prometheus/prometheus v1.8.2-0.20211214150951-52c693a63be1
```